apps/consulting/posts/improvements-to-bsr-matrix-multiplication-in-pytorch.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Improvements to BSR Matrix Multiplication in PyTorch"
+published: October 1, 2023
+author: andrew-james
+description: 'Quansight engineers have implemented several new kernels for matrix multiplication
+involving sparse and dense arguments. These implementations, available in PyTorch 2.1,  offer a
+significant performance improvement over those available in previous versions of PyTorch. In
+addition, they have the unique property that they are implemented using `triton`, however they are
+not the product of `torch.compile`.'
+category: [Open Source Software, PyTorch, Triton, GPU]
+---
+<base target="_blank" />
+
+Quansight engineers have implemented several new kernels for matrix multiplication involving sparse
```

Comment:
@amjames @lezcano Are these sole contribs, or also team effort with Meta?

If all are one or the other, we'll be able to say it once at the top. Otherwise we'll want to be specific at each contrib.

(I haven't read anything yet, so you may have already done this.)